app/controllers/site_controller.rb

```diff
@@ -81,7 +81,13 @@ def load_charts_data
       }
     end
 
-    top_five_protected_areas = StaticStat.where(habitat: @habitat).order("protected_percentage DESC").pluck(:country_id, :protected_percentage).to_a.first(5)
+    top_five_country_ids = top_five_countries.map do |country|
+      iso3 = country.first
+      country_id = Country.find_by(iso3: country.first).id
```

Comment:
You could remove the unused `iso3` variable or use that in the next line

Reply:
Many thanks!